src/Command.js

```diff
@@ -14,6 +14,7 @@ export default class Command {
     this.repository = new Repository();
     this.progressBar = progressBar;
     this.logger = logger;
+    this.concurrency = (!flags || flags.concurrency === undefined) ? 4 : flags.concurrency;
```

Comment:
I think it is good that it would throw, however I think me might need to add a test to make sure it doesn't accidentally get refactored away the next time somebody reads over that code?!